src/main/java/com/gulteking/mqttbackendserver/serviceimpl/MqttTopicServiceImpl.java

```diff
@@ -28,21 +28,34 @@ public List<MqttTopics> findAll() {
         return mqttTopicsRepository.findAll();
     }
 
+    @Override
+    public Boolean findAllByIsConnected(Boolean isConnected) {
+        try {
+            List<MqttTopics> topicsList = mqttTopicsRepository.findAll();
+            for (MqttTopics mqttTopics : topicsList) {
+                mqttSubscriber.publishMessage(mqttTopics.getMtSerialNumber(), new Gson().toJson(mqttTopics));
```

Comment:
can we use publisher instead of subscriber here.
let subscriber do the job of subscription and publisher do the job of publishing messages.